parts/k8s/kubernetesinstalls.sh

```diff
@@ -21,6 +21,47 @@ function installEtcd() {
     fi
 }
 
+function netfilterFix() {
```

Comment:
this looks like it would fit better in https://github.com/Azure/acs-engine/blob/master/parts/k8s/kubernetesconfigs.sh